sdbootutil

```
@@ -701,6 +772,9 @@ show_entry()
 
 list_snapshots()
 {
+	if [ $no_snapshots -eq 1 ]; then
+		err "list-snapshots incompatible with --no-snapshots!"
+	fi
```

just silently do nothing?

---

I'm not a big fan of doing things completely silent, but a warning might be better in this case.

---

Added.